.github/workflows/build-container.yml

```
@@ -25,7 +25,7 @@ jobs:
           OS: ${{ matrix.os }}
         run: |
           PREFIX="output_image=${GITHUB_REPOSITORY}:${SERIES}"
-          if [[ $OS == 'el8' ]]; then
+          if [[ $OS == 'el9' ]]; then
```

This change should change the "no os" tag from el8 to el9. Is that the desired behavior or should it be kept on el8?

---

```suggestion
          if [[ $OS == 'el8' ]]; then
```

Hrm yeah, I don't think we can pull the trigger on this OS update just yet.